assets/css/style.css

```diff
@@ -0,0 +1,234 @@
+body {
+    background-color: #d9dcd6;
+    background-image: url('../images/grapes.jpg');
+    background-repeat: no-repeat;
+    background-size: cover;
+    min-height: 100vh;
+}
+
+
+main {
+    display: flex;
+    justify-content: center;
+}
+
+header{
+    background-color: #13293df1;
+    min-height: 10vh;
+    box-shadow: 0 0 2.5px 2px;
+
+}
+
+header h1 {
+    color: white;
+    display: flex;
+    justify-content: center;
+    padding-top: 1%;
+}
+
+#headerBar {
+    display: flex;
+    justify-content: space-evenly;
+    font-family: 'Courier New';
+}
+
+#questionCard h1 {
+    display: flex;
+    text-align: center;
+}
+
+header a,p{
+    color: white;
+    font-size: 115%;
+}
+
+header a{
+    padding-top: 1rem;
+}
+
+
+.container {
+    max-width: 70%;
+    min-width: 70%;
+    background: rgb(254, 250, 238);
+    min-height: 68vh;
+    max-height: 68vh;
+    border-radius: 15px;
+    padding: 1vh;
+    box-shadow: 0 0 9px;
+    margin-top: 6.5rem;
+    font-size: 125%;
+}
+
+
+.container h1 {
+    display: flex;
+    justify-content: center;
+    font-size: 225%;
+    padding-top: 5rem;
+    font-family: 'Times New Roman', Times, serif;
```

Comment:
You could make things look a lot better with a simple font change. Times just doesn't look that great. Check out Google Fonts for a ton of free fonts. 

https://fonts.google.com/